### PR TITLE
Add Kannada language files. Also, add Amitabh Bachchan as third actor in Hindi.

### PR DESCRIPTION
**Pull request by GitTallMan**
The following are the changes proposed:
1. Kannada language has to be included.
2. Amitabh Bachchan has to be added as third actor in Hindi.

### CODE REVIEW

**Comment by GitTallMan** [comment]
@casey-collab: Please, review the changes made.

File: language.md [hunk 1]
```diff
@@ -67,3 +67,5 @@ Ak hovoríš po slovensky, [klikni sem](slovak/marsmelaci.md)
 اگر به زبان فارسی صحبت می‌کنید، [به اینجا بروید](persian/otagheh-basteh.md)
 
 आप हिंदी बोलते हैं   ?,[go here](hindi/bollywood.md)
+
+ನೀವು ಕೆಚ್ಚೆದೆಯ ಕನ್ನಡಿಗರಾದರೆ, [ಇಲ್ಲಿ ಒತ್ತಿ](kannada/romaanchanavee_kannada.md).
```
Comment (casey-collab) [inline language.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: kannada/soochike.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+ಇದು ಕನ್ನಡ ಬರಹಗಳ ಸೂಚಿಕೆ.
```
Comment (casey-collab) [inline kannada/soochike.md]: It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by GitTallMan** [comment]
The link to the new file has been added in my most recent commit. @casey-collab, kindly confirm.

File: language.md (hunk 1 above)
Comment (casey-collab) [inline language.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: kannada/soochike.md (hunk 2 above)
Comment (casey-collab) [inline kannada/soochike.md]: It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by GitTallMan** [comment]
@casey-collab: Fixed the broken link issue. Kindly confirm and merge.

**Comment by casey-collab** [comment]
Looks great! :ribbon:

**Pull request merged**